.gitignore

```diff
@@ -13,3 +13,5 @@ dist
 
 # Data files
 /cities/data/*
+# Common virtualenv directory name
+env
```

Comment:
I would say that `venv` is probably a more common one, but I'm not really interested in including every potential common virtualenv directory for all developers.

Reply:
Should I remove it or can we keep it?

Reply:
I would just remove it, since it's a local configuration choice and not generally applicable to all django-cities developers. And even [Django itself](https://github.com/django/django/blob/master/.gitignore) does not include virtualenv directory/ies in its `.gitignore`.